website/addons/dryad/static/dryadNodeConfig.js

```diff
@@ -0,0 +1,375 @@
+'use strict';
+
+var ko = require('knockout');
+require('knockout.punches');
+var Raven = require('raven-js');
+var $osf = require('js/osfHelpers');
+//This is for creating the new folder, the license information and
+//moving everything over
+var waterbutler = require('js/waterbutler');
+
+function ViewModel(url) {
```

Comment:
Consider `oop.extend`ing this from `OauthAddonFolderPicker`; it should give you some generic functionality out of the box, and also make it easier to provide a more consistent UX. See recent changes to `s3NodeConfig.js`.